setup.py

```diff
@@ -90,3 +91,22 @@ def long_description():
         'mypy_extensions>=0.4',
     ],
 )
+
+
+print(in_color('green', '\nZulip Terminal was successfully installed!'))
+if MACOS:
+    sys.exit(0)
+print('\nTo ensure notifications are displayed, you may wish to run the '
+      'following:\n')
+if WSL:
+    print(in_color('blue', 'Run powershell as "admin" and run these commands'
+                   ' to install dependencies for showing notifications:'))
+    print(in_color('red', '\nset-executionpolicy remotesigned\nInstall-Module'
+                          ' -Name BurntToast\n'))
+elif LINUX:
+    print(in_color('blue', 'The following command installs `notify-send` on'
+                   ' debain based systems, similar commands can be founds for'
```

Comment:
Same typos ;) Having the text in only one place is good.

Could you add a note to the issue for generating the README partially from keys.py, as this seems related?

Reply:
Still 'Debain'.